lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -761,13 +834,12 @@ def draw_rubberband(self, x0, y0, x1, y1):
 
 
 class ToolbarGTK3(ToolContainerBase, Gtk.Box):
-    def __init__(self, toolmanager):
+    def __init__(self, toolmanager, flow='horizontal'):
```

Comment:
The `flow` keyword is only available for MEP22 classes, how do you handle that when using MEP27 without MEP22?